cdc/cdc/owner/owner.go

```diff
@@ -199,6 +199,14 @@ func (o *Owner) Tick(stdCtx context.Context, rawState orchestrator.ReactorState)
 	return state, nil
 }
 
+// Close try to close the owner when the owner is not running.
+// Note: This method must not be called with `Tick`. Please be careful.
+func (o *Owner) Close(ctx cdcContext.Context) {
```

Comment:
Suggest to name as `CloseAllCaptures`, as we are not closing `Owner` here.

[Here](https://github.com/tikv/migration/blob/73d43788dfc2513d5bc3e579363438f147ccfb7d/cdc/cdc/owner/owner.go#L191) has similar code block, suggest to reuse this function.

(And maybe we also need the wrapper of context.)

At last, we need to double check about the thread-safety.